docs/tutorials/elasticsearch.ipynb

```diff
@@ -0,0 +1,750 @@
+        "\n",
+        "Once the data is available in the elasticsearch cluster, only `tensorflow-io` is required to stream the data into the models.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA"
+      },
+      "source": [
+        "### Install the required tensorflow-io and elasticsearch packages"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "48B9eAMMhAgw"
+      },
+      "outputs": [],
+      "source": [
+        "print(\"Installing the tensorflow-io-nightly package !\")\n",
+        "!pip install tensorflow-io-nightly\n",
+        "\n",
+        "print(\"Installing the elasticsearch package !\")\n",
+        "!pip install elasticsearch\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gjrZNJQRJP-U"
+      },
+      "source": [
+        "### Import packages"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "m6KXZuTBWgRm"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "from subprocess import Popen, PIPE, STDOUT\n",
+        "import time\n",
+        "from sklearn.model_selection import train_test_split\n",
+        "from elasticsearch import Elasticsearch\n",
+        "import numpy as np\n",
+        "import pandas as pd\n",
+        "import tensorflow as tf\n",
+        "from tensorflow.keras import layers\n",
+        "from tensorflow.keras.layers.experimental import preprocessing\n",
+        "import tensorflow_io as tfio"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "eCgO11GTJaTj"
+      },
+      "source": [
+        "### Validate tf and tfio imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "dX74RKfZ_TdF"
+      },
+      "outputs": [],
+      "source": [
+        "print(\"tensorflow-io version: {}\".format(tfio.__version__))\n",
+        "print(\"tensorflow version: {}\".format(tf.__version__))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW"
+      },
+      "source": [
+        "## Download and setup the Elasticsearch instance\n",
+        "\n",
+        "For demo purposes, the open-source version of the elasticsearch package is used.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YUj0878jPyz7"
+      },
+      "outputs": [],
+      "source": [
+        "!wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.9.2-linux-x86_64.tar.gz\n",
+        "!wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.9.2-linux-x86_64.tar.gz.sha512\n",
+        "!shasum -a 512 -c elasticsearch-oss-7.9.2-linux-x86_64.tar.gz.sha512 \n",
```

Comment:
`!` commands don't cancel execution if they fail.

If you use `%%bash`, and make the `shasum` the last command in the cell, then it will stop execution on failure.

Reply:
addressed in the commit